server/channels/app/session.go

```diff
@@ -18,6 +18,9 @@ import (
 	"github.com/mattermost/mattermost/server/v8/channels/store"
 )
 
+// MaxSessionsLimit prevents a potential DOS caused by creating an unbounded number of sessions; MM-55320
+const MaxSessionsLimit = 500
```

Comment:
The user agent limit is not exported, but this one is. I'd prefer this to be not exported as well.